README.md

```diff
@@ -176,38 +194,37 @@ If it's found, the unattended installation will begin and source that profile as
 - [Desktop](https://github.com/archlinux/archinstall/tree/master/archinstall/default_profiles/desktops)
 - [Server](https://github.com/archlinux/archinstall/tree/master/archinstall/default_profiles/servers)
 
-The profiles' definitions and the packages they will install can be directly viewed in the menu, or
-[default profiles](https://github.com/archlinux/archinstall/tree/master/archinstall/default_profiles)
-
+The profiles' definitions and the packages they will install can be directly viewed in the menu, or [default profiles](https://github.com/archlinux/archinstall/tree/master/archinstall/default_profiles)
 
 # Testing
 
+There's also a [Building and Testing](https://github.com/archlinux/archinstall/wiki/Building-and-Testing) guide.<br>
+
 ## Using a Live ISO Image
 
 If you want to test a commit, branch, or bleeding edge release from the repository using the standard Arch Linux Live ISO image,
 replace the archinstall version with a newer one and execute the subsequent steps defined below.
 
-*Note: When booting from a live USB, the space on the ramdisk is limited and may not be sufficient to allow
-running a re-installation or upgrade of the installer. In case one runs into this issue, any of the following can be used
-- Resize the root partition https://wiki.archlinux.org/title/Archiso#Adjusting_the_size_of_the_root_file_system
-- The boot parameter `copytoram=y` (https://gitlab.archlinux.org/archlinux/mkinitcpio/mkinitcpio-archiso/-/blob/master/docs/README.bootparams#L26)
-can be specified which will copy the root filesystem to tmpfs.*
+* Note: When booting from a live USB, the space on the ramdisk is limited and may not be sufficient to allow running a re-installation or upgrade of the installer. In case one runs into this issue, any of the following can be used
+   * Resize the root partition https://wiki.archlinux.org/title/Archiso#Adjusting_the_size_of_the_root_file_system
+   * The boot parameter `copytoram=y` (https://gitlab.archlinux.org/archlinux/mkinitcpio/mkinitcpio-archiso/-/blob/master/docs/README.bootparams#L26) can be specified which will copy the root filesystem to tmpfs.*
 
+### Tips
```

Comment:
These aren't tips but rather steps to accomplish using the Live USB testing and the 2 points above are prerequisites